build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -942,6 +945,17 @@ message ExecutedActionMetadata {
   // When the worker completed executing the action command.
   google.protobuf.Timestamp execution_completed_timestamp = 8;
 
+  // The amount of time the worker spent executing the action command,
+  // not including overheads before or after execution such as
+  // marshalling inputs/outputs.
+  //
+  // The server SHOULD avoid including time spent the client doesn't
+  // have control over, and MAY reduce the execution duration to account
+  // for delays that occur during execution itself (e.g., lazily loading
+  // data from the Content Addressable Storage, live migration of
+  // virtual machines, emulation overhead).
```

Comment:
Maybe also add something to the effect of `SHOULD be <= execution_start_timestamp - execution_completed_timestamp.` ?

Reply:
We could, but is this something we want to limit explicitly? We're mentioning the term 'emulation' in the description, so actually executing something faster than wall time is not impossible, especially for low-end embedded targets.

Reply:
Right, makes sense.